.github/workflows/k8s-kind-network-e2e.yaml

```diff
@@ -24,6 +17,29 @@ env:
   cluster_name: cilium-testing
 
 jobs:
+  check_changes:
```

Comment:
If this is a new job and it generates some output, then presumably the next job needs to somehow reference / depend on this job name + output variable name? I don't see that part currently.

Reply:
Yes! :duh: